feg/gateway/sbi/sbi.go

```diff
@@ -66,7 +67,11 @@ type BaseClientWithNotifier struct {
 }
 
 func NewEchoServer() *EchoServer {
-	return &EchoServer{echo.New()}
+	e := echo.New()
+	if glog.V(2) {
+		e.Use(ServerLoggingMiddleware())
+	}
+	return &EchoServer{e}
```

Comment:
awesome! 
This is where we had to get: just one place to add the logger and forget about it